language_version_metadata/Python3X__0__metadata.json

```diff
@@ -0,0 +1,9 @@
+[{
```

Comment:
shouldn't we default to python 36 here and have all 4

Reply:
these are here just for backward compatibility.